charts/kube-prometheus-stack/templates/grafana/configmaps-datasources.yaml

```diff
@@ -24,6 +24,7 @@ data:
 {{- if .Values.grafana.sidecar.datasources.defaultDatasourceEnabled }}
     - name: Prometheus
       type: prometheus
+      uid: promeheus
```

Comment:
I guess it should be "prometheus" with "t" missing. And also in the second uid below

Reply:
Thx! gixed